### ISSUE TITLE
To-do list: 

### ISSUE THREAD

**Issue by zwanli** [zwanli/atrnn-prototype#2]
- [x] UPGRADE to tensorflow v1.3 and adapt the code accordingly 
- [ ] Partial run

-  Text prepossessing
   - [ ] Check unknown words
      - 13000 unknown words out of 40000 !! 
      - numbers
      - [ ] Include the abstracts when training words embedding 
   - [ ] remove less frequent words
   - [x] Padding
        - [x] Figure out why the queue is not outputting everything 
   - [x] Sequence length
   - [x] load pre-trained word embedding
   - [x] parse cite_u_like_a
   - [x] parse cite_u_like_t

- Network architecture 
   - [ ] dropout layer
   - [ ] hyper parameters 
   - [ ] cost function

- Input data 
   - [ ] Passing previous hidden states:
      - [x]  Bi-directional layer
      - [ ]  second layer 
    - [x] Random batching
    - [x] Train,  test splitting following Wang strategy 

- Metrics
   - [x] Recall
   - [x] Recall@
   - [x] ndcg 

- [x] Run on alacran
